src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -22,6 +22,37 @@ public static int multiply(int a1, int a2) {
         if (point1 == point) {
             System.out.println("Equal");
         }
+
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "a1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1226)